CRM/Mailing/BAO/Mailing.php

```diff
@@ -2437,6 +2437,22 @@ public static function del($id) {
    * @throws CRM_Core_Exception
    */
   public static function self_hook_civicrm_pre(\Civi\Core\Event\PreEvent $event) {
+    if ($event->action === 'create') {
+      $params = &$event->params;
```

Comment:
@eileenmcnaughton was it this line?

Reply:
@colemanw nope - the next one down & that whole block

Reply:
@colemanw  these are not php 7.3 compliant `??=`

Reply:
Oh, sorry I misread. I thought you were talking about 8.3 not 7.3.
I see, so the problem is that 5.69 was supposed to support 7.3 because we [didn't do the version bump until 5.70](https://github.com/civicrm/civicrm-core/pull/28583).
@seamuslee001